apps/website/src/components/Playground/template/App.js

```diff
@@ -12,12 +12,30 @@ export default function App() {
   const [rules, setRules] = React.useState('');
   const ref = React.useRef(null);
   React.useEffect(() => {
+    const styleTag = document.createElement('style');
+    styleTag.id = 'playground';
+    document.head.append(styleTag);
+
     /** @type import('@griffel/core').GriffelRenderer */
     const playgroundRenderer = {
       id: 'playground',
-      insertCSSRules(cssRules) {
-        const raw = Object.values(cssRules)
-          .reduce((acc, val) => acc.concat(val), [])
+      insertCSSRules(cssRuleBuckets) {
+        const cssRules = Object.values(cssRuleBuckets)
+          .reduce((acc, val) => {
+            return acc.concat(val);
+          }, [])
+          .filter(Boolean);
+
+        cssRules.forEach(cssRule => {
+          styleTag.sheet?.insertRule(cssRule);
+        });
+
+        if (!styleTag.sheet) {
+          return;
+        }
+
+        const raw = Array.from(styleTag.sheet.cssRules)
```

Comment:
```suggestion
        const raw = Array.from(styleTag.sheet.cssRules).reverse()
```

Can we add `.reverse()` so it will follow the definition order?

![image](https://user-images.githubusercontent.com/14183168/160432705-bbe8c855-d411-47ab-b35b-fc8c5fc99421.png)

Reply:
yeah I didn't notice that, good idea